src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -55,7 +56,7 @@
   private String petSAEmail;
 
   // Google credentials for the user
-  private UserCredentials userCredentials;
+  private GoogleCredentials googleCredentials;
```

Comment:
nit: add a comment here explaining that this can be either the end-user google creds or pet SA google creds pulled from app-default